backends/arm/test/test_arm_baremetal.sh

```diff
@@ -101,7 +101,7 @@ test_pytest_models() { # Test ops and other things
     source backends/arm/scripts/install_models_for_test.sh
 
     # Run arm baremetal pytest tests without FVP
-    pytest  --verbose --color=yes --durations=0 backends/arm/test/models
+    pytest  --verbose --color=yes --numprocesses=auto --durations=0 backends/arm/test/models
```

Comment:
I always assumed it would do this automatically after reading `worker` related messages. but this is good, I didn't realize `--dist=no` by default.